dynamic_rest/fields/fields.py

```diff
@@ -49,17 +51,36 @@ def to_representation(self, value):
     def to_internal_value(self, data):
         return data
 
+    def get_meta_arg(self, local_kwarg, meta_attr):
```

Comment:
Might be reasonable to default `meta_attr` to `local_kwarg + '_fields'` as that seems to be the convention